index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>TicTacToe</title>
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+<body>
+<h1>Tic Tac Toe</h1>
+<div id="wrapper">
+<span id="upperS">
+	<div class="sq" id="one">
+		?
+	</div>
+	<div class="sq" id="two">
+		?
+	</div>
+	<div class="sq" id="three">
+		?		
+	</div>
+
+</span>
+<span id="upperS">
+	<div class="sq" id="four">
+		?
+	</div>
+	<div class="sq" id="five">
+		?
+	</div>
+	<div class="sq" id="six">
+		?
+	</div>
+
+</span>
+<span id="upperS">
+	<div class="sq" id="seven">
+		?
+	</div>
+	<div class="sq" id="eight">
+		?
+	</div>
+	<div class="sq" id="nine">
+		?
+	</div>
+
+</span>
+<button id = "clearB" type="reset" value ="reset">Clear</button>
```

Comment:
Even though your clear button works, I don't recommend leaving any spaces in HTML tags. Conventional syntax would look like the following:

```
<button id="clearB" type="reset" value="reset">Clear</button>
```